config-prettyfaces/src/main/java/com/ocpsoft/pretty/PrettyContext.java

```diff
@@ -95,9 +95,8 @@ protected PrettyContext(final HttpServletRequest request)
       requestURL = requestURL.decode();
 
       requestQuery = QueryString.build(request.getQueryString());
-
-      log.trace("Initialized PrettyContext");
-      log.info("Initialized PrettyContext (has support for Jakarta EE9+)");
+      
+      log.trace("Initialized PrettyContext (has support for Jakarta EE9+)");
```

Comment:
It looks like this was introduced as part of the EE9 update. Let's just put this back to the way it was.

`log.trace("Initialized PrettyContext");`

It doesn't need to say anything else.

Reply:
Done as requested.